### ISSUE TITLE
For projects we should be able to deploy plugins to specific groups and call them when needed.

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#2949]
The title might be confusing, but it's actually pretty self explanatory. I can make you understand better in a simple scenario. For example, I am constantly working on different fields such as c++, python, web, mobile, flutter, devops. For all my projects, vscode is my biggest helper and because I can't use plugins correctly, I can't use vscode at full performance.

You can say install the baseline plugins for each project or workspace!. Why!!! Why should I add category-like plugins manually instead of optionally adding them to projects?

To summarize briefly, as a simple method
You can do a nice development that we can synchronize to our microsoft or github account as `[{"categoryName1":["extension1",extension2"...],...}]` And we can import category based add-on feature from .vscode/settings.
